cypress/integration/plugins/alerting-dashboards-plugin/composite_level_monitor_spec.js

```diff
@@ -180,9 +180,8 @@ describe('CompositeLevelMonitor', () => {
 
       // Wait for monitor to be created
       cy.wait('@updateMonitorRequest').then(() => {
-        cy.get('.euiTitle--large').contains(
-          `${SAMPLE_VISUAL_EDITOR_MONITOR}_edited`
-        );
+        cy.wait(5000);
+        cy.contains(`${SAMPLE_VISUAL_EDITOR_MONITOR}_edited`);
```

Comment:
Maybe `timeout` option will work?
```suggestion
        cy.contains(`${SAMPLE_VISUAL_EDITOR_MONITOR}_edited`, {timeout: 5000});
```

Reply:
@amsiglan does the above suggestion not work?

Reply:
It should also work for me